fontir/src/orchestration.rs

```diff
@@ -137,33 +97,62 @@ impl Context {
             .unwrap();
         let buf_io = BufWriter::new(raw_file);
         serde_yaml::to_writer(buf_io, &content)
-            .map_err(|e| panic!("Unable to serialize to {:?}: {}", file, e))
+            .map_err(|e| panic!("Unable to serialize\n{:#?}\nto {:?}: {}", content, file, e))
             .unwrap();
     }
 
+    fn restore<V>(&self, file: &Path) -> V
+    where
+        V: ?Sized + DeserializeOwned,
+    {
+        let raw_file = File::open(file)
+            .map_err(|e| panic!("Unable to read {:?} {}", file, e))
+            .unwrap();
```

Comment:
if you just `unwrap` this error, you should get approximately the same information as you are manually printing here.

Reply:
Hm, I thought it didn't tell me the file but maybe that's a figment of my imagination. Will double-check

Reply:
https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=963503e29a7d3b50eee0f1c5e3cd0e6e gives me `Os { code: 2, kind: NotFound, message: "No such file or directory" }` which does not include the path